dept44-scheduler/src/main/java/se/sundsvall/dept44/scheduling/config/SchedulingConfiguration.java

```diff
@@ -0,0 +1,45 @@
+package se.sundsvall.dept44.scheduling.config;
+
+import javax.sql.DataSource;
+import net.javacrumbs.shedlock.core.LockProvider;
+import net.javacrumbs.shedlock.provider.jdbctemplate.JdbcTemplateLockProvider;
+import net.javacrumbs.shedlock.spring.annotation.EnableSchedulerLock;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.EnableAspectJAutoProxy;
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.scheduling.annotation.EnableScheduling;
+
+/**
+ * Configuration for scheduling and locking tasks.
+ * <p>
+ * Uses {@link JdbcTemplateLockProvider} for locking tasks.
+ * <p>
+ * Locks tasks for at most 2 minutes.
+ * <p>
+ * Uses {@link EnableSchedulerLock} for locking tasks.
+ *
+ * @see JdbcTemplateLockProvider
+ * @see EnableSchedulerLock
+ */
+@Configuration
```

Comment:
Does this config class need to be defined in the file: `org.springframework.boot.autoconfigure.AutoConfiguration.imports` ?